manim/mobject/svg/svg_mobject.py

```diff
@@ -124,28 +125,28 @@ def _ensure_valid_file(self):
         if self.file_name is None:
             raise Exception("Must specify file for SVGMobject")
 
-        if os.path.exists(self.file_name):
+        if Path(self.file_name).exists():
             self.file_path = self.file_name
             return
 
-        relative = os.path.join(os.getcwd(), self.file_name)
-        if os.path.exists(relative):
+        relative = Path.cwd().joinpath(self.file_name)
+        if Path(relative).exists():
             self.file_path = relative
             return
 
         possible_paths = [
-            os.path.join(config.get_dir("assets_dir"), self.file_name),
-            os.path.join(config.get_dir("assets_dir"), self.file_name + ".svg"),
-            os.path.join(config.get_dir("assets_dir"), self.file_name + ".xdv"),
-            self.file_name,
-            self.file_name + ".svg",
-            self.file_name + ".xdv",
+            Path(config.get_dir("assets_dir")).joinpath(self.file_name),
```

Comment:
use the `/` operator here: https://docs.python.org/3/library/pathlib.html#operators

Reply:
Also `config.get_dir` returns a path object and hence casting is also not required.

Reply:
Fixed!